common/presenters/move-to-card-component.test.js

```diff
@@ -94,19 +94,19 @@ describe('Presenters', function() {
             expect(transformedResponse.tags).to.deep.equal({
               items: [
                 {
-                  href: '/move/12345#concealed_items',
+                  href: '/move/12345#Concealed items',
```

Comment:
feels like we should be using fragments without spaces. Or is this just me?

Reply:
Have updated this so that it will always use kebab case for fragments.